extensions/amp-story/1.0/amp-story-reaction-quiz.css

```diff
@@ -0,0 +1,447 @@
+ }
+ 
+ .i-amphtml-story-reaction-post-selection.i-amphtml-story-reaction-has-data[dir=rtl] .i-amphtml-story-reaction-quiz-option:nth-of-type(4)::before {
+     transform: translateX(calc(-1 * var(--option-4-percentage))) !important;
+ }
+ 
+ .i-amphtml-story-reaction-post-selection.i-amphtml-story-reaction-has-data .i-amphtml-story-reaction-quiz-option::before {
+     left: -101% !important;
+     width: 100% !important;
+     height: 100% !important;
+     border-radius: 0px !important;
+     background: var(--reaction-theme-shading) !important;
+     color: var(--reaction-theme-shading) !important;
+     transition: transform 1s !important;
+     border: solid 2px !important;
+ }
+ 
+ .i-amphtml-story-reaction-post-selection.i-amphtml-story-reaction-has-data[dir=rtl] .i-amphtml-story-reaction-quiz-option::before {
+     left: 101% !important;
+ }
+ 
+ .i-amphtml-story-reaction-post-selection.i-amphtml-story-reaction-has-data .i-amphtml-story-reaction-option-selected[correct].i-amphtml-story-reaction-quiz-option {
+     background: var(--correct-color) !important;
+ }
+ 
+ .i-amphtml-story-reaction-post-selection.i-amphtml-story-reaction-has-data .i-amphtml-story-reaction-option-selected:not([correct]).i-amphtml-story-reaction-quiz-option {
+     background: var(--incorrect-color) !important;
+ }
+ 
+ .i-amphtml-story-reaction-post-selection .i-amphtml-story-reaction-option-selected.i-amphtml-story-reaction-quiz-option {
+     /* Ups the padding by one to account for the border which must be removed
+      * to allow the shading to reach the edge of the box */
+      padding: 9px 17px 9px 9px !important;
+      border: none !important;
+ }
+ 
+ .i-amphtml-story-reaction-post-selection[dir=rtl] .i-amphtml-story-reaction-option-selected.i-amphtml-story-reaction-quiz-option {
+     /* Switches the extra padding to the left from the right, since the answer choice and the percentage switch */
+     padding: 9px 9px 9px 17px !important;
+     border: none !important;
+ }
+ 
+ .i-amphtml-story-reaction-post-selection.i-amphtml-story-reaction-has-data .i-amphtml-story-reaction-option-selected[correct].i-amphtml-story-reaction-quiz-option::before {
+     background: var(--correct-color-shaded) !important;
+     color: var(--correct-color-shaded) !important;
+ }
+ 
+ .i-amphtml-story-reaction-post-selection.i-amphtml-story-reaction-has-data .i-amphtml-story-reaction-option-selected:not([correct]).i-amphtml-story-reaction-quiz-option::before {
+     background: var(--incorrect-color-shaded) !important;
+     color: var(--incorrect-color-shaded) !important;
+ }
+ 
+ /**
+  * Proper bounce animation scale values calculated in a CodePen
+  * using methodology from Medium post "Spring Animation in CSS"
+  * by Thai Pangsakulyanont.
+  *
+  * https://codepen.io/jackbsteinberg/pen/ExaZpjZ
+  * https://medium.com/@dtinth/spring-animation-in-css-2039de6e1a03
+  */
+ @keyframes answer-choice-bounce {
```

Comment:
If we want to use this animation for other components we could move it to the reaction.css file, but not until we have the animations defined for the other components

Reply:
SGTM 👍